pkg/operation/botanist/component/vpnauthzserver/external_authz_server.go

```diff
@@ -64,6 +68,7 @@ type authzServer struct {
 	namespace           string
 	imageExtAuthzServer string
 	replicas            int32
+	kubernetesVersion   string
```

Comment:
Make it `*semver.Version`?

Reply:
This is also not consistent across components, and `defaultVPNAuthzServer` was using the string and I just passed it down. I can change this if you want, of course.

Reply:
I would vote for it since it simplifies the code a little bit, WDYT?